packages/cli/src/__tests__/commands/Studio.test.ts

```diff
@@ -24,49 +21,6 @@ function sendRequest(message: any): Promise<any> {
 
 let studio: Studio
 
-describe('Studio CLI', () => {
-  beforeEach(() => {
-    // Back to original env vars
-    process.env = { ...originalEnv }
-    // Update env var because it's the one that is used in the schemas tested
-    process.env.STUDIO_URL = 'prisma://aws-us-east-1.prisma-data.com/?api_key=MY_API_KEY'
-    process.env.STUDIO_DIRECT_URL = 'postgres://localhost:1234'
-  })
-
-  afterEach(() => {
-    // Back to original env vars
-    process.env = { ...originalEnv }
-  })
-
-  it('should fail if url is prisma://', async () => {
```

Comment:
This should not fail any more.

Reply:
This should still be the case, I think. If you don't provide a `directUrl` then it should fail.